### PR TITLE
Phase2-hgx360R Inhibit ZDC digitization in Phase2 scenario

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Inhibit ZDC digitization in Phase2 scenario

#### PR validation:

Use the runTheMatrix test workflows

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by abdoulline** [comment]
@bsunanda   I'm afraid you didn't read my email...   
- you didn't address Andrea's comment about [a spurious line in the config](https://github.com/cms-sw/cmssw/pull/46720/files#diff-9091b3d3d57231cae37783f979339ce4d112c12890164a0b328261e400dd7545R72)
- we need not only inhibit ZDC digitization for phase2, but also [fix the hardcoded conditions of it](https://github.com/cms-sw/cmssw/compare/master...abdoulline:cmssw:ZDC_Digi_off_Phase2?expand=1)

File: SimCalorimetry/HcalSimProducers/python/hcalUnsuppressedDigis_cfi.py [hunk 1]
```diff
@@ -64,7 +64,10 @@
 
 # remove HE processing for phase 2, completely put in HGCal land
 from Configuration.Eras.Modifier_phase2_hgcal_cff import phase2_hgcal
-phase2_hgcal.toModify(hcalSimBlock, killHE = cms.bool(True) )
+phase2_hgcal.toModify(hcalSimBlock,
+                      doZDCDigi = cms.bool(False),
+                      killHE = cms.bool(True)
```
Comment (perrotta) [inline SimCalorimetry/HcalSimProducers/python/hcalUnsuppressedDigis_cfi.py:69]:
```suggestion
                      doZDCDigi =False,
                      killHE = True
```

File: SimCalorimetry/HcalSimProducers/python/hcalUnsuppressedDigis_cfi.py [hunk 2]
```diff
@@ -64,7 +64,10 @@
...
+)
 
 # remove HE processing for phase 2, completely put in HGCal land
```
Comment (perrotta) [inline SimCalorimetry/HcalSimProducers/python/hcalUnsuppressedDigis_cfi.py:72]: Please update also this comment, which was just copy-pasted from the previous one but it does not correspond to what actually implemented here

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by perrotta** [comment]
Thank you @bsunanda , this is fine with me

**Comment by civanch** [comment]
+1

**Comment by mandrenguyen** [comment]
+1

**Pull request merged**